docs/topics_5/web-analyze-apps.adoc

```diff
@@ -3,17 +3,17 @@
 [id='using_web_console_analyze_apps_{context}']
 = Analyzing applications with the {WebName}
 
-In order to use the {WebName} to analyze applications, you must create a project. Each project is a way to group applications for analysis and specifies the settings to use during the analysis of its applications. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
+To use the {WebName} to analyze applications, you must create one or more projects. Each project is a way to group applications for analysis and specifies the settings to use during the analysis of its applications. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
 
 Follow these steps to use the {WebName} to analyze your applications and review the results.
 
```

Comment:
Colon after "review the results", not a period, because you are introducing an ordered list.